src/mods/chat.js

```diff
@@ -849,7 +849,7 @@ idrinth.chat = {
             idrinth.chat.join ( data.data );
             return;
         }
-        idrinth.core.alert ( idrinth.text.get ( "chat.message.loginFailMoment" ) );
+        idrinth.core.alert ( idrinth.text.get ( "chat.error.loginMoment" ) );
```

Comment:
Moment is likely unneeded, please have a look if it can be combined with chat.error.login